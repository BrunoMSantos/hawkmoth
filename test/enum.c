/**
 * Enum doc.
 */
enum foo {
	/**
	 * Enumeration comment.
	 */
	bar,
	/**
	 * Another.
	 */
	baz,
};
