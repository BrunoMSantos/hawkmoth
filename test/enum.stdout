
.. c:type:: enum foo

   Enum doc.


   .. c:macro:: bar

      Enumeration comment.


   .. c:macro:: baz

      Another.

